config.go

```diff
@@ -10,6 +10,13 @@ import (
 	"github.com/segmentio/parquet-go/compress"
 )
 
+type PageReadMode int
+
+const (
+	PageReadModeAsync PageReadMode = iota
+	PageReadModeSync
```

Comment:
I imagine that the intent is to maintain the existing behavior by having async the default.

Thinking about what default behavior would create the least number of surprises (and maintenance work answering Github issues), I wonder if using sync as the default would be a better choice here.

Reply:
Agree. I was tempted to choose Sync, but thought backwards compat was a good choice. After we work out the questions above I can change the default to Sync.

Reply:
swapped default to sync